doc/source/examples/ale_regression_california.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<p>I would say that the deciles don't show where there isn't any data, but is more like an indication of lower density. Take for example the plot of <code>MedInc</code>. In the last decile there exist data points, it's just that there is lower density of points.  For the example below, the statement might actually be true. But the absence of points I think is illustrated by the absence of markers on the lineplot, and not by the deciles.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Agreed, rephrased.</p>